samza-core/src/main/scala/org/apache/samza/coordinator/JobCoordinator.scala

```diff
@@ -121,8 +121,9 @@ object JobModelManager extends Logging {
                         changelogManager: ChangelogPartitionManager,
                         localityManager: LocalityManager,
                         streamMetadataCache: StreamMetadataCache,
-                        streamPartitionCountMonitor: StreamPartitionCountMonitor) = {
-    val jobModel: JobModel = initializeJobModel(config, changelogManager, localityManager, streamMetadataCache)
+                        streamPartitionCountMonitor: StreamPartitionCountMonitor,
+                        containerIds: java.util.List[Integer]) = {
```

Comment:
Fyi, I am going to rename this file : JobCoordinator.scala -> JobModelManager.scala. There shouldn't be any changes in it.